tests/foreman/api/test_errata.py

```diff
@@ -173,6 +175,38 @@ def _fetch_available_errata(self, host, expected_amount, timeout=120):
                 )
             )
 
+    @upgrade
+    @tier3
+    def test_positive_bulk_install_package(self):
+        """Bulk install package to a collection of hosts
+
+        :id: c5167851-b456-457a-92c3-59f8de5b27ee
+
+        :Steps: PUT /api/v2/hosts/bulk/install_content
+
+        :expectedresults: package is installed in the hosts.
+
+        :BZ: 1528275
+
+        :CaseLevel: System
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as client:
+            client.install_katello_ca()
+            client.register_contenthost(
+                self.org.label, self.activation_key.name)
+            self.assertTrue(client.subscribed)
+            client.enable_repo(REPOS['rhst7']['id'])
+            client.install_katello_agent()
+            host_id = entities.Host().search(query={
+                'search': 'name={0}'.format(client.hostname)})[0].id
+            self._install_package(
+                [client],
+                [host_id],
+                FAKE_1_CUSTOM_PACKAGE_NAME,
+                via_ssh=False,
+                rpm_package_name=FAKE_2_CUSTOM_PACKAGE
```

Comment:
Is this extra kwarg really necessary? Can't you just pass 'walrus' as `package_name`? That will install latest version and ensure it's installed. Or what's the expected flow here?

Reply:
@abalakh we need to be sure it has install a package with version and arch, for REX bulk install  we pass "walrus" as package name but for verification we need full package name with version and arch "rpm_package_name" that is "walrus-5.21-1.noarch"

Reply:
@abalakh also note that there is no other test that use that part of the function eg: via_ssh=False, except the test implemented here.